source/beamline/AMPVControl.cpp

```diff
@@ -459,8 +483,8 @@ void AMPVwStatusControl::onReadPVInitialized()
 }
 
 void AMPVwStatusControl::onWritePVInitialized() {
-	lowLimitValue_ = writePV_->lowerControlLimit();
-	highLimitValue_ = writePV_->upperControlLimit();
```

Comment:
I'm not sure I understand what you mean here, @dretrex. The low and high limit values look like they're set when the write PV reports as initialized.